packages/swap/src/event-handlers/swapScheduled.ts

```diff
@@ -26,6 +26,8 @@ const swapScheduledArgs = z.object({
   origin: z.union([depositChannelSwapOrigin, vaultSwapOrigin]),
   swapType: swapTypeSchema,
   brokerCommission: u128.optional(),
+  // >= v1.4.0
```

Comment:
should we add a similar comment for `brokerCommission`?

Reply:
done